mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/processing/impl/PropertyBindingProcessor.java

```diff
@@ -28,8 +29,16 @@
 	@Override
 	public void process(PropertyMappingStep mapping, PropertyBinding annotation,
 			PropertyMappingAnnotationProcessorContext context) {
-		PropertyBinder binder = createBinder( annotation.binder(), context );
-		mapping.binder( binder );
+		PropertyBinderRef propertyBinder = annotation.binder();
+		PropertyBinder binder = createBinder( propertyBinder, context );
+
+		if ( propertyBinder.params() != null ) {
+			Map<String, Object> params = ParameterAnnotationsReader.toMap( propertyBinder.params() );
```

Comment:
What's the point of exposing a method in the `context` if you don't use it? :)

```suggestion
			Map<String, Object> params = context.toMap( propertyBinder.params() );
```